### PR TITLE
Function Locate

### PR DESCRIPTION
**Pull request by TheAbsolutionism**
### Description
This PR aims to add the ability to locate functions by providing the script it's in as well as the line number. 
Useful for big projects or dev teams that need to find a function.

---
**Target Minecraft Versions:** any 
**Requirements:** none 
**Related Issues:** none

### CODE REVIEW

File: src/main/java/ch/njol/skript/lang/function/Signature.java [hunk 1]
```diff
@@ -173,8 +173,8 @@ public int getMinParameters() {
 	public int hashCode() {
 		return name.hashCode();
 	}
-
-	@Override
+  
+  	@Override
```
Comment (Efnilite) [inline src/main/java/ch/njol/skript/lang/function/Signature.java:177]: does this use spaces?

File: src/main/java/ch/njol/skript/SkriptCommand.java [hunk 2]
```diff
@@ -84,7 +88,8 @@ public class SkriptCommand implements CommandExecutor {
 			.add("changes")
 			.add("download")
 		).add("info"
-		).add("help");
+		).add("help"
+		).add("locate");
```
Comment (Efnilite) [inline src/main/java/ch/njol/skript/SkriptCommand.java:92]:
```suggestion
		.add("help")
		.add("locate");
```

File: src/main/java/ch/njol/skript/SkriptCommand.java [hunk 3]
```diff
@@ -84,7 +88,8 @@ public class SkriptCommand implements CommandExecutor {
 			.add("changes")
 			.add("download")
 		).add("info"
```
Comment (Efnilite) [inline src/main/java/ch/njol/skript/SkriptCommand.java:90]:
```suggestion
		).add("info")
```

**Review by Efnilite — APPROVED** [review]

File: src/main/resources/lang/turkish.lang [hunk 4]
```diff
@@ -44,6 +44,7 @@ skript command:
 		info: Skript'in alias ve dökümantasyon bağlantılarını gösterir.
 		gen-docs: Eklenti klasöründeki şablonları kullanarak bir dokümantasyon oluşturur.
 		test: Skript'in dahili testleri için kullanılır.
+		locate: Bir işlevi bulmak için kullanılır
```
Comment (erenkarakal) [inline src/main/resources/lang/turkish.lang:47]: Bir fonksiyonu bulmak için kullanılır.

File: src/main/resources/lang/turkish.lang [hunk 5]
```diff
@@ -115,6 +116,10 @@ skript command:
 		server: Sunucu sürümü: <aqua>%s
 		addons: Skript addon'ları: <aqua>%s
 		dependencies: Bağımlılıklar: <aqua>%s
+	locate:
+		function found: Fonksiyon '&e%s&f' şurada bulunur: &6%s&f, Satır: &6%s
+		function provided: İşlev '&e%s&f' tarafından sağlanır Skript
```
Comment (erenkarakal) [inline src/main/resources/lang/turkish.lang:121]: Fonksiyon '&e%s&f' Skript tarafından eklendi.

File: src/main/resources/lang/turkish.lang [hunk 6]
```diff
@@ -115,6 +116,10 @@ skript command:
...
+		function not found: İşlev '%s' bulunamıyor
```
Comment (erenkarakal) [inline src/main/resources/lang/turkish.lang:122]: '%s' isimli fonksiyon bulunamadı.

**Review by erenkarakal — CHANGES_REQUESTED** [review]

**Pull request closed without merging**